tools/assets-automation/assets-maintenance-tool/Azure.Sdk.Tools.Assets.MaintenanceTool.Tests/CLITests.cs

```diff
@@ -0,0 +1,59 @@
+global using NUnit.Framework;
+using Azure.Sdk.Tools.Assets.MaintenanceTool.Model;
+using Azure.Sdk.Tools.Assets.MaintenanceTool.Scan;
+using Microsoft.AspNetCore.Http;
+using Newtonsoft.Json;
+
+namespace Azure.Sdk.Tools.Assets.MaintenanceTool.Tests
+{
+    public class CLITests
+    {
+        public string TestDirectory { get; protected set; } 
+
+        [SetUp]
+        public void Setup()
+        {
+            var workingDirectory = Path.Combine(Path.GetTempPath(), Guid.NewGuid().ToString());
+
+            if (!Directory.Exists(workingDirectory))
+            {
+                Directory.CreateDirectory(workingDirectory);
+            }
+
+            // copy our static test files there
+            var source = Path.Combine(Directory.GetCurrentDirectory(), "TestResources");
+            var target = Path.Combine(workingDirectory, "TestResources");
+
+            Microsoft.VisualBasic.FileIO.FileSystem.CopyDirectory(source, target);
```

Comment:
Do we have to use `Micorosft.VisualBasic` ?

Reply:
Yeah for some reason this easy `CopyDirectory` implementation isn't available in standard dotnet path. Given that:

- We don't have an additional external dependency added to make this available
- This is a test function

I didn't see the point in writing this up myself.